src/styles/components/framework-configuration/styles.less

```diff
@@ -37,3 +37,11 @@
 .glyphicon-plus {
   font-style: normal;
 }
+
+.package-last-updated-danger {
```

Comment:
anything speaking against just reusing `.text-danger` here?

Reply:
Since the design wasn't finished, I wanted to make it easy to add new styles.

Reply:
that makes a lot of sense! thanks for sharing! 💡

Reply:
Maybe that's overengineering,  and won't be needed.